README.md

```diff
@@ -20,4 +20,9 @@ To contribute:
     git push origin feat/short-desc-abt-the-feature
     ```
 
-4. Create a PR to the master branch
\ No newline at end of file
+4. Create a PR to the master branch
+
+5. Run it with this code
+    ```bash
+    flask run
+    ```
```

Comment:
Could you please also add the flask db init, flask db migrate and upgrade commands?

Thanks